### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by rudis**
Fixes a few minor typos/inconsistencies and adds installation instructions for Stretch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -508,7 +510,7 @@ updating the instruction directly.  This approach also ensures that the code
 modification path is reliable, since ftrace has been doing it successfully for
 years.
 
-**Q Is kpatch compatible with \<insert kernel debugging subsystem here\>?**
+**Q. Is kpatch compatible with \<insert kernel debugging subsystem here\>?**
```
Comment (joe-lawrence) [inline README.md]: Can you also correct the line starting with "**Q: Is it possible to register a function that..." a few lines down.  (ie, `s/:/.` )

File: README.md [hunk 2]
```diff
@@ -351,6 +351,7 @@ can have some major pitfalls if you're not careful.  To learn more about how to
 properly create live patches, see the [Patch Author
 Guide](doc/patch-author-guide.md).
 
+
```
Comment (joe-lawrence) [inline README.md]: I don't think these spaces show up in the Markdown rendering, but if we want to be consistent with two blank lines, then can you also update the spaces preceding the Installation and License sections.

File: README.md [hunk 3]
```diff
@@ -405,6 +406,7 @@ The primary steps in kpatch-build are:
 - Link all the output objects into a cumulative object
 - Generate the patch module
 
+
 ### Patching
```
Comment (joe-lawrence) [inline README.md]: I guess the pattern is to double-space the ### sub-sections?  If so, does the Debian 7.x Install subsection need an extra line, too?

Comment (rudis) [reply README.md]: I tried to follow the pattern in the document and that seemed to be the most consistent way. The Debian 7.x section has four levels (####) so I added only one empty line there.

File: README.md (hunk 2 above)
Comment (rudis) [reply README.md]: Will do.

File: README.md (hunk 1 above)
Comment (rudis) [reply README.md]: Will do.

**Comment by rudis** [comment]
Pushed updated version which addresses the mentioned issues. Also added another commit to use https:// for youtube links.

**Comment by joe-lawrence** [comment]
👍  looks good, thanks @rudis

**Pull request merged**